README.md

```
@@ -2,27 +2,60 @@
+   ```bash
+   git clone <repository-url>
+   cd resumeX
+   ```
+
+2. Install dependencies:
+   ```bash
+   pip install -r requirements.txt
+   ```
+
+3. Create a `.env` file in the project root and add your Gemini API key:
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)